src/schema/rules/datatypes/meg.yaml

```diff
@@ -60,8 +66,10 @@
 - suffixes:
   - headshape
   extensions:
-  - .*
+  - "*"
```

Comment:
I think this was a merge conflict resolution? Seems out of scope.

```suggestion
  - .*
```

Reply:
Ah, yes. I think I just merged it badly.